src/marketplace/types/marketplace_accounts.ts

```diff
@@ -28,3 +28,10 @@ export interface OrderSide {
     Buy,
     Sell
 }
+
+export enum GmLogs {
+    CancelOrderMemo,
+    ExchangeMemo,
+    InitializeMemo,
+    RegisterCurrencyMemo,
```

Comment:
Also - this should resolve string values. Right now they're integer values (default for Typescript enums):

```suggestion
    CancelOrderMemo = 'CancelOrderMemo',
    ExchangeMemo = 'ExchangeMemo',
    InitializeMemo = 'InitializeMemo',
    RegisterCurrencyMemo = 'RegisterCurrencyMemo',
```